### ISSUE TITLE
Term Request: Map-like corneal epithelial opacities

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11737]
**Preferred term label:**
Map-like corneal epithelial opacities

**Synonyms**
* Geographic corneal intraepithelial opacities
* Map lines of cornea
* Geographic subepithelial maps

**Definition (free text, please give PubMed ID)**
Irregular, circumscribed, geographic or map-like areas of thickened, gray, hazy epithelium with scalloped borders located centrally or paracentrally. Histologically, they represent intraepithelial extensions and multilamellar sheets of aberrant basal laminar material (PMID: 38359414).

**Parent term (use hpo.jax.org/app)**
Abnormal corneal epithelium morphology (HP:0011495)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
* Epithelial basement membrane dystrophy (EBMD) (ORPHA:98956, OMIM:121820)
* LCD variant p.Leu509Arg (LCD-L509R) (OMIM:608471)

**Your nano-attribution (ORCID)**
0000-0001-9042-0508
